src/Game.ts

```diff
@@ -96,7 +96,7 @@ export class Game {
         // and store it
         this.serverId = localStorage.getItem("server") ?? "";
         if (this.serverId === "") {
-            this.serverId = uuidv4();
+            this.serverId = createServerId();
```

Comment:
Might want to reset any server IDs that have already been generated, so if they're longer than X characters - replace.

Reply:
At this stage I'd just replace any long ones, the game has only been around a week. No one is going to mind at this stage.